function_documentation.rst

```diff
@@ -0,0 +1,200 @@
+===================
+Database functions
+===================
+
+
+- *function name* = **get_artist_by_id**:
```

Comment:
You don't need to say function name in front of each function, it can be removed